src/modules/UI/scenes/TransactionList/TransactionList.ui.js

```diff
@@ -265,77 +179,71 @@ export class TransactionList extends Component<Props, State> {
-                          </T>
-                        )}
+          <View style={styles.balanceBoxContents}>
+            {!isAccountBalanceVisible ? (
+              <View style={[styles.totalBalanceWrap]}>
+                <View style={[styles.hiddenBalanceBoxDollarsWrap]}>
+                  <T style={[styles.currentBalanceBoxHiddenText]}>{SHOW_BALANCE_TEXT}</T>
+                </View>
+              </View>
+            ) : (
+              <View style={styles.balanceShownContainer}>
+                <View style={[styles.iconWrap]}>
+                  {logo ? (
+                    <Image style={[{ height: '100%' }]} source={{ uri: logo }} resizeMode={'cover'} />
+                  ) : (
+                    <T style={[styles.request]}>{displayDenomination.symbol}</T>
+                  )}
+                </View>
+                <View style={[styles.currentBalanceBoxBitsWrap]}>
+                  <View style={{ flexDirection: 'row' }}>
+                    {displayDenomination.symbol ? (
+                      <T numberOfLines={1} style={[styles.currentBalanceBoxBits, styles.symbol]}>
+                        {displayDenomination.symbol + ' '}
+                        <T numberOfLines={1}>{cryptoAmountString}</T>
+                      </T>
+                    ) : (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {cryptoAmountString}
+                      </T>
+                    )}
 
-                        {!displayDenomination.symbol && (
-                          <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
-                            {' ' + selectedCurrencyCode}
-                          </T>
-                        )}
-                      </View>
-                    </View>
-                    <View style={[styles.currentBalanceBoxDollarsWrap]}>
-                      <T numberOfLines={1} style={[styles.currentBalanceBoxDollars]}>
-                        {fiatBalanceString}
+                    {!displayDenomination.symbol && (
+                      <T numberOfLines={1} style={styles.currentBalanceBoxBits}>
+                        {' ' + selectedCurrencyCode}
                       </T>
-                    </View>
-                  </View>
-                ) : (
-                  <View style={[styles.balanceHiddenContainer]}>
-                    <T style={[styles.balanceHiddenText]}>{SHOW_BALANCE_TEXT}</T>
-                  </View>
-                )}
-              </TouchableOpacity>
-              <View style={[styles.requestSendRow]}>
-                <TouchableHighlight style={[styles.requestBox, styles.button]} underlayColor={styleRaw.underlay.color} onPress={Actions.request}>
-                  <View style={[styles.requestWrap]}>
-                    <Image style={{ width: 25, height: 25 }} source={requestImage} />
-                    <T style={[styles.request]}>{REQUEST_TEXT}</T>
+                    )}
```

Comment:
what does this change do?

Reply:
Change in the structure of the JSX of the balance box to better create uniformity between the "Show Balance" version and the actual Balance version.